test/unit/query-generator/remove-column-query.test.ts

```diff
@@ -0,0 +1,33 @@
+import { DataTypes } from '@sequelize/core';
+import { buildInvalidOptionReceivedError } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/check.js';
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+describe('QueryGenerator#removeColumnQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    age: DataTypes.INTEGER,
+  }, { timestamps: false });
+
+  it('generates a DROP COLUMN query in supported dialects', () => {
+    const sql = queryGenerator.removeColumnQuery(User.tableName, 'age');
+
+    expectsql(() => sql, {
+      default: `ALTER TABLE [Users] DROP COLUMN [age];`,
+      postgres: `ALTER TABLE "public"."Users" DROP COLUMN "age";`,
+      snowflake: `ALTER TABLE "Users" DROP "age";`,
+      sqlite: 'CREATE TABLE IF NOT EXISTS `Users_backup` (`0` a, `1` g, `2` e);INSERT INTO `Users_backup` SELECT `0`, `1`, `2` FROM `Users`;DROP TABLE `Users`;ALTER TABLE `Users_backup` RENAME TO `Users`;',
+      'mariadb mysql': 'ALTER TABLE `Users` DROP `age`;',
```

Comment:
```suggestion
      'mariadb mysql snowflake': `ALTER TABLE [Users] DROP [age];`,
      sqlite: 'CREATE TABLE IF NOT EXISTS `Users_backup` (`0` a, `1` g, `2` e);INSERT INTO `Users_backup` SELECT `0`, `1`, `2` FROM `Users`;DROP TABLE `Users`;ALTER TABLE `Users_backup` RENAME TO `Users`;',
```

Reply:
Weirdly enough if I do this

```js
'mariadb mysql snowflake': `ALTER TABLE [Users] DROP [age];`
```
It is not able to replace the `[` with backticks `

Reply:
That's strange. @ephys do you know why?

Reply:
@WikiRik I think its because of [this](https://github.com/sequelize/sequelize/blob/main/test/support.ts#L431)
We are only doing it for `expectations.default`

Reply:
That seems to be the case indeed. Then we'll leave this for now and we can fix it in a future refactor